src/lib/Libwin/passwd.c

```diff
@@ -4817,15 +4817,15 @@ logon_pw(char *username,
 	char msg[4096])
```

Comment:
Seems very strange to supply the array size as part of the parameter. Is this required in Windows? I would expect msg to be declared as a character pointer.